go-controller/pkg/ovn/pods.go

```diff
@@ -184,7 +186,13 @@ func getRoutesGatewayIP(pod *kapi.Pod, gatewayIPnet *net.IPNet) ([]util.PodRoute
 			routes = append(routes, route)
 		}
 	} else {
-		gatewayIP = gatewayIPnet.IP
+		if namespaceExternalGW == "" {
```

Comment:
@rcarrillocruz @nerdalert I think we need to have the Cluster/Service networks as explicit routes, while the default route for everything off-cluster goes to the external gateway. Eg we do need the block right above:

https://github.com/ovn-org/ovn-kubernetes/blob/cc5ecb2701b0d7d720ef8ff2aec3e1c92240c282/go-controller/pkg/ovn/pods.go#L175

Reply:
eg shouldn't the external gateway case *also* set `otherDefaultRoute`?